segtypes/n64/codesubsegment.py

```diff
@@ -310,6 +310,9 @@ def add_labels(self, funcs):
 
                 if insn[0].mnemonic != "branch" and insn[0].mnemonic.startswith("b") or insn[0].mnemonic.startswith("j"):
                     indent_next = True
+            
+            if addsuffix:            
```

Comment:
I think it would make sense to just do the asm_endlabels check here rather than pass it through as a function arg

Reply:
I only wanted endlabels to be generated for asm files and not nonmatchings, because asm processor doesn't handle the suffix correctly.

Reply:
Ohhh okay. makes sense